src/ctree-app/ctree-landing.html

```diff
@@ -511,45 +508,39 @@ <h1 align="left">Participate</h1>
         </div>
         <div id="footer" class="section-styling">
           <div class="table-row">
-            <div class="col-5 table-cell line">
+            <div class="col-12 table-cell">
               <div class="inner-row-1">
                 <div id="social-media-icon-bar">
                   <ul>
-                    <li><a href="https://twitter.com/"><i class="twitter"></i><img src="[[rootPath]]/images/twitter-icon-32.png"></a></li>
-                    <li><a href="https://www.facebook.com/"><i class="facebook"</i><img src="[[rootPath]]/images/facebook-icon-32.png"></a></li>
-                    <li><a href="https://github.com/F4IF/ctree-demo"><i class="github"></i><img src="[[rootPath]]/images/github-icon-32.png"></a></li>
-                    </ul>
+                    <li><a href="https://github.com/F4IF/ctree-demo"><i class="github"></i><img src="[[rootPath]]/images/github.png"></a></li>
+                    <li><a href="https://www.slack.com"><i class="slack"</i><img src="[[rootPath]]/images/slack.png"></a></li>
+                    <li><a href="https://www.facebook.com"><i class="facebook"</i><img src="[[rootPath]]/images/facebook.png"></a></li>
+                    <li><a href="https://twitter.com"><i class="twitter"></i><img src="[[rootPath]]/images/twitter.png"></a></li>
+                  </ul>
                 </div>
               </div>
               <div class="inner-row-2">
-                <div id="ctree-blurb">
-                  <p>The Collaboration Tree is a project of the Foundation For an Innovative Future</p>
-                </div>
+                <div class="flex-grid">
+                    <div class="flex-row">
+                      <div class="flex-column">
+                        <div id="ctree-blurb">
+                          <p>The Collaboration Tree is a project of the Foundation For an Innovative Future</p>
+                        </div>
+                      </div>
+                      <div class="flex-column">
+                        <div id="copyright">
+                          <p>&copy Foundation For an Innovative Future.</p>
```

Comment:
I never noticed this part of the comp.  It's actually an `@` not a copyright symbol, though I can see why you'd think that.  I can double check with Alex but the idea was probably to indicate it's a project of the larger nonprofit, maybe with a link to the nonprofit website (innovativeFuture.org).